src/Workspaces/Core/Portable/Differencing/Match.LongestCommonSubsequence.cs

```diff
@@ -17,12 +17,16 @@ internal LongestCommonSubsequence(Match<TNode> match)
                 _match = match;
             }
 
-            protected override bool ItemsEqual(IReadOnlyList<TNode> oldSequence, int oldIndex, IReadOnlyList<TNode> newSequence, int newIndex)
+            protected override bool ItemsEqual(
+                IReadOnlyList<TNode> oldSequence, int oldIndex, 
+                IReadOnlyList<TNode> newSequence, int newIndex)
             {
                 return _match.Contains(oldSequence[oldIndex], newSequence[newIndex]);
             }
 
-            internal Dictionary<TNode, TNode> GetMatchingNodes(IReadOnlyList<TNode> oldNodes, IReadOnlyList<TNode> newNodes)
+            internal Dictionary<TNode, TNode> GetMatchingNodes(
```

Comment:
The lines were not visible for me on github.  That's generally when i wrap so that i don't need to vertically scroll when reading things here.